server/app/services/PartService.ts

```diff
@@ -43,7 +44,10 @@ export class PartService {
   }
 
   public async createPart(body: IPart): Promise<IPart> {
-    await this.handleMaterialStockUpdates(body.stock, body.type);
+    const price = await this.handleMaterialStockUpdates(body.stock, body.type);
+    body.costPrice = price;
+    // 30% profit from cost price
+    body.sellingPrice = price * 1.3;
```

Comment:
I feel like margin should be set by the user in the UI

Reply:
up for debate i guess

Reply:
sure, that could be done.